src/pokemon_storage_system.c

```diff
@@ -6197,7 +6221,7 @@ static void SaveCursorPos(void)
     sSavedCursorPosition = sCursorPosition;
 }
 
-static u8 GetSavedCursorPos(void)
+u8 GetSavedCursorPos(void)
```

Comment:
What's the point of this no longer being static if it isn't used outside of this file?

Reply:
Ah, this can be changed back to static. I was calling this in other files before I set the cursor position to `gSpecialVar_0x8004`